app/parse.py

```diff
@@ -1,4 +1,11 @@
+import requests
 from dataclasses import dataclass
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://mate.academy/"
```

Comment:
Ensure that `BASE_URL` points to the specific page containing the course information. The current URL may not contain the necessary HTML elements for parsing.